app/src/main/java/com/jerboa/ui/components/post/PostListing.kt

```diff
@@ -1380,7 +1381,7 @@ private fun ThumbnailTile(
             postView.post.thumbnail_url?.also { thumbnail ->
                 PictrsThumbnailImage(
                     thumbnail = thumbnail,
-                    blur = blurNSFW && nsfwCheck(postView),
+                    blur = needNsfwBlur(blurNSFW.toEnum(), postView.community.nsfw, postView.post.nsfw),
```

Comment:
Add override for postview that calls this basically. And remove all occurances of this